src/sonic-config-engine/data/l2switch.j2

```diff
@@ -6,8 +6,11 @@
         {%- if ns.firstPrinted %},{% endif %}
 
         "{{ key }}": {
-            "alias": "{{ value.alias }}",
-            "lanes": "{{ value.lanes }}",
+            {%- for keyPort,valuePort in value.items() %}
```

Comment:
>, [](start = 27, length = 1)

Add a blank after `,`. Also some more occurrences in the whole file.

Reply:
Fixed